src/includes/getting-started-primer/apple.mdx

```diff
@@ -0,0 +1,23 @@
+<Note>
+
+*Sentry's SDKs report an error automatically whenever a thrown exception goes uncaught in your application causing the application to crash. Sentry's iOS, tvOS, and watchOS SDK hook into all signal and exception handlers, except for MacOS.*
+
+*The SDK builds a crash report that persists to disk and tries to send the report right after the crash. Since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.*
+
+</Note>
+
+**Features:**
```

Comment:
Should we also add offline caching to the list of features? Same applies to Android.

Reply:
I think better to mention in the list, yes. (and I'll also add it to Android).